core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala

```diff
@@ -395,8 +395,8 @@ class StandaloneDynamicAllocationSuite
     }
 
     var apps = getApplications()
-    // kill executor 1
-    assert(sc.killExecutor(executors.head))
+    // kill executor 1, and actually nothing to kill
+    assert(!sc.killExecutor(executors.head))
```

Comment:
if so, we should not kill excutors.head(27). it should be excutor(1). am I right?

���� Grace from mobile phone

-------- �������� --------
������Re: [spark] [SPARK-9552] Return "false" while nothing to kill in killExecutors (#9796)
��������andrewor14
��������apache/spark
������"Huang, Jie"


In core/src/test/scala/org/apache/spark/deploy/StandaloneDynamicAllocationSuite.scala<https://github.com/apache/spark/pull/9796#discussion_r45414046>:

> @@ -395,8 +395,8 @@ class StandaloneDynamicAllocationSuite
>      }
>
>      var apps = getApplications()
> -    // kill executor 1
> -    assert(sc.killExecutor(executors.head))
> +    // kill executor 1, and actually nothing to kill
> +    assert(!sc.killExecutor(executors.head))


no, the idea is more like the following:

  *   you start with executors {27, 28}
  *   you kill and replace 27, so you end up with executors {28, 29}
  *   now you want to kill 28, this should succeed (but currently it doesn't in the tests)

��
Reply to this email directly or view it on GitHub<https://github.com/apache/spark/pull/9796/files#r45414046>.